_benchmark/reference/commands/command-flags.md

```diff
@@ -290,4 +290,34 @@ The name and path used for the chart's output. Default is `stdout`.
 ## limit
 <!-- vale on -->
 
-Limits the number of search results for recent test runs. Default is `10`.
\ No newline at end of file
+Limits the number of search results for recent test runs. Default is `10`.
+
+<!-- vale off -->
+## latency-percentiles
+<!-- vale on -->
+
+Specifies a list of latency percentiles to report after the workload runs. Default is `50,90,99,99.9,99.99,100`. 
+
+<!-- vale off -->
+## throughput-percentiles
+<!-- vale on -->
+
+Specifies a list of throughput percentiles to report after the workload runs, in addition to mean/median/max/min. Default is `None`. 
+
+<!-- vale off -->
+## randomization-enabled
+<!-- vale on -->
+
+Enables randomization of values in range queries, where the values are drawn from standard value functions registered with `register_standard_value_source` in the workload's workload.py. Default is `False`. 
```

Comment:
Nit: For folks who are not familiar, it would be good to add a small blurb defining "standard value functions".

Reply:
What happens if there is no such function registered?  Please indicate that an error is raised if that is the case.

Reply:
standard value functions can be defined as:
```
It's a function you call to generate a random pair of values for a certain field.
```
Source: https://github.com/opensearch-project/opensearch-benchmark/pull/455#issuecomment-1928366750

@peteralfonsi feel free to add more insight if needed. Govind has a good point, we should also address what happens if no function is registered.

Reply:
Yes, I will add a definition and the case when nothing is registered.